src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -120,15 +121,15 @@ If you need to change the default privacy settings, you can choose specific clas
   >
       If you want to customize exactly what is masked with asterisks, you have a few options:
 
-        * On the <DoNotTranslate>**Application settings**</DoNotTranslate> page, add your own CSS selectors or check the masking boxes.
+        * On the <DoNotTranslate>**Application settings**</DoNotTranslate> page, add your own CSS selectors or check the masking boxes. Note that using mask selectors will **not** mask user input, so if you need to hide user input, we recommend using block selectors. 
```

Comment:
@CherylFrankenfield Can you confirm this description is correct?

Reply:
Masking selectors do not affect user input. This is correct.